### PR TITLE
Prepare release 1.0.0

### PR DESCRIPTION
**Pull request by petrdvorak**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -122,19 +107,14 @@
         <dependency>
             <groupId>com.sun.xml.messaging.saaj</groupId>
```
Comment (romanstrobl) [inline pom.xml:108]: Are the libraries related to Web Services still required? If you don't use the old SOAP interface, I think we can remove them.

**Review by romanstrobl — CHANGES_REQUESTED** [review]
Overall looks ok, let's just check that whether we still need all the SOAP-related libraries for Java 11 after migration to REST.

File: pom.xml (hunk 1 above)
Comment (petrdvorak) [reply pom.xml:108]: Good point. We do not need this and we decrease the size of a resulting WAR file from 71.7MB to 48.8MB by removing these dependencies. 👍

**Review by romanstrobl — APPROVED** [review]
Looks good

**Pull request merged**